generator/src/main/java/com/scottlogic/deg/generator/inputs/ProfileReader.java

```diff
@@ -17,11 +19,20 @@
 import java.util.stream.Collectors;
 
 public class ProfileReader {
-    public Profile read(Path filePath) throws IOException, InvalidProfileException {
+    public Profile read(Path filePath, boolean doProfileValidation) throws IOException, InvalidProfileException {
```

Comment:
maybe rename parameter from `doProfileValidation` to `validateProfile`.

Either that or pass in the `ProfileValidationReporter` that should be used, having the `NoopProfileValidationReporter` type passed in when validation isn't required. The latter may produce a better implementation as switching isn't performed in the reader, only in the `Generate`/`GenerateTestCases` _controller_ types

Reply:
changed to pass in the reporter to use - defaults to noop.